src/helpers/external.js

```diff
@@ -77,23 +77,18 @@ const setToggleCallback = (selector, callback) => {
     }
 
     if (isAccordion) {
-
         $('[data-toggle-target]', container)
             .filter(toggle => !toggle.hasAttribute('data-toggle-accordion-exclude'))
             .forEach(toggle => {
                 toggle.addEventListener('click', () => {
                     callback.call(this, toggle);
```

Comment:
you can bind the callback for consolidation of the call methods e.g.

```js
const call = callback.bind(this)
...
call(toggle)
```

thus below

```js
call(container)
```

Reply:
Feel free to submit a PR if you feel this would be beneficial 🍯